src/subclass/widget.rs

```diff
@@ -506,11 +506,11 @@ impl<T: WidgetImpl + ObjectImpl> WidgetImplExt for T {
             let data = self.get_type_data();
             let parent_class = data.as_ref().get_parent_class() as *mut gtk_sys::GtkWidgetClass;
             if let Some(f) = (*parent_class).dispatch_child_properties_changed {
-                let pspecs_ptr = pspecs
+                let mut pspecs_array = pspecs
                     .iter()
                     .map(|p| p.to_glib_none().0)
-                    .collect::<Vec<_>>()
-                    .as_mut_ptr();
+                    .collect::<Vec<_>>();
+                let pspecs_ptr = pspecs_array.as_mut_ptr();
```

Comment:
IMHO `.map(|p| p.to_glib_none().0)` leak too
and better use something like this
```
let stashes = ... .map(|p| p.to_glib_none()).collect...();
let pspecs_ptr = stashes.0.as_mut_ptr();
```

Reply:
FWIW valgrind doesn't show this leaking.